src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -61,7 +71,7 @@ private void AssertLicenseLimits(string type, ServerStore serverStore, DatabaseR
                 if (maxReplicationFactorForSharding == null && multiNodeSharding)
                     return;
 
-                if (CanAssertLicenseLimits(context, minBuildVersion: MinBuildVersion60000) == false)
+                if (CanAssertLicenseLimits(context, new List<int>(MinBuildVersion60000)) == false)
```

Comment:
Let's stay with a single version.
And start checking this from the next build version.